packages/twenty-server/src/modules/activity/standard-objects/activity.workspace-entity.ts

```diff
@@ -40,20 +51,25 @@ export class ActivityWorkspaceEntity extends BaseWorkspaceEntity {
 
   @WorkspaceField({
     standardId: ACTIVITY_STANDARD_FIELD_IDS.body,
-    type: FieldMetadataType.TEXT,
+    type: FieldMetadataType.RICH_TEXT,
     label: 'Body',
     description: 'Activity body',
-    icon: 'IconList',
+    icon: 'IconFilePencil',
+    defaultValue: "''",
```

Comment:
Is it the default value we actually want in case of a RICH_TEXT (saying that before it seems to be a nested json in most cases). Shoulnd't it be null?